### ISSUE TITLE
problem with bicolor broken boxes

### ISSUE THREAD

**Issue by nignasi** [T-F-S/tcolorbox#19]
When a bicolor box is broken a possible filling color for the lower part (segmentation style) stops at first page and becomes gray in following fragments. The color for the upper part (interior style) is correct even in broken boxes.

[example.pdf](https://github.com/T-F-S/tcolorbox/files/794513/sensetitol-1.pdf)

`\documentclass[a5paper]{article}
\usepackage[most]{tcolorbox}
\usepackage{geometry}
\usepackage{lipsum}

\newtcolorbox{mybox}[1][]{%
	enhanced, 
	breakable, 
	bicolor,
	interior style={blue!30},
	segmentation style={red!30},
	#1
}

\begin{document}

\begin{mybox}
This is the upper part
\tcblower
\lipsum[3-8]
\end{mybox}

\begin{mybox}
\lipsum[1-4]
\tcblower
\lipsum[3-5]
\end{mybox}

\end{document}`

**Comment by T-F-S** [comment]
You are right, the `segmenation style` is ignored for several cases with `bicolor`.

The example can be rewritten using
```latex
  colback=blue!30,
  colbacklower=red!30,
  %interior style={blue!30},
  %segmentation style={red!30},
```
to get a correct result.

But I consider this `segmenation style` behaviour as a bug, and I will provide some bug fix in the near future.

**Comment by T-F-S** [comment]
Preview version 4.02pre1 (2017/02/23) with fixes is on GitHub now.

**Comment by T-F-S** [comment]
Fixed with 4.02
